### DISCUSSION TITLE
Thank You for this Marvelous Project!

### DISCUSSION THREAD

**Discussion by ciur** [mantinedev/mantine#6753] [Kind words]
I recently migrated my open source project from tw bootstrap to Mantine. 
I was worried because of tw bootstrap lack of updates. All alternatives for react based components UI which I've found were (semi) commercial, or at least with restrictive license.

I've started experimenting with Mantine couple of months ago.
Now I am relieved: working with it is a pleasure: project is well documented, easy to work with and there is a wide range of components and utils.

Mantine's MIT license (+ regular updates + great documentation + wide range of components) makes it obvious choice when you need
to pick UI library for open source projects.

Thank you for your work!

**Comment by rtivital** [comment]
Thanks for kind words!
